contracts/AccountManager.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.4;
+
+contract AccountManager {
+  
+  mapping(address => uint) balances;
+  function AccountManager() {
+    balances[tx.origin] = 0;
```

Comment:
Why do you use `tx.origin` instead of `msg.sender` ?

Reply:
Also this line is not needed.

Reply:
I wanted to be sure who the account was coming from. Know now that it is less flexible than msg.sender. But yeah you are right, I should just delete that line since it initializes to zero anyways.